app/src/main/java/org/wikipedia/readinglist/AddToReadingListDialog.kt

```diff
@@ -90,7 +90,7 @@ open class AddToReadingListDialog : ExtendedBottomSheetDialogFragment() {
             displayedLists.clear()
             displayedLists.addAll(readingLists)
             if (!showDefaultList && displayedLists.isNotEmpty()) {
-                displayedLists.removeAt(0)
```

Comment:
The "first" item in the list of reading lists is not necessarily the default list. We should use a more proper way of addressing the default list, instead of assuming it exists at position 0.